components/konflux-ui/staging/base/proxy/nginx.conf

```diff
@@ -161,6 +166,39 @@ http {
             include /mnt/nginx-generated-config/bearer.conf;
         }
 
+        # GET requests to the following endpoints are handled from the namespace-lister.
+        # * /api/k8s/api/v1/namespaces
+        # * /api/k8s/api/v1/namespaces/
+        #
+        # Requests with other methods are handled by the Kube-API
+        location ~* ^/api/k8s/api/v1/namespaces(/?)$ {
+            try_files $uri @$ns_target;
+        }
+
+        location @namespacelister {
+            auth_request_set $email  $upstream_http_x_auth_request_email;
+            auth_request /oauth2/auth;
+            proxy_read_timeout 30m;
+            proxy_set_header X-Email $email;
+
+            rewrite ^.*$ /api/v1/namespaces break;
```

Comment:
instead of re writing the entire path to a constnat, you can add add the path to the url in the proxy_pass directive.

Reply:
right, let me apply this change